usecases/configuration_test.go

```diff
@@ -207,9 +207,6 @@ func createConfigNoTraits(sys *components.System, assetAmount int) (err error) {
 	return
```

Comment:
Lines 135-152: See if you can skip the singular setup case (and just use the for loop).